docs/source/changelog.rst

```diff
@@ -6,7 +6,7 @@ Jupyter notebook changelog
 A summary of changes in the Jupyter notebook.
 For more detailed information, see `GitHub <https://github.com/jupyter/notebook>`__.
 
-.. _release-4.1.x:
```

Comment:
I think in the long run 4.1.x is enough, we can keep 4.1.0 also, and have 4.1.1 related news in under 4.1.0 link.